engine/gameobject/src/gameobject/comp_script.cpp

```diff
@@ -223,11 +224,14 @@ namespace dmGameObject
         RunScriptParams run_params;
         run_params.m_UpdateContext = params.m_UpdateContext;
         CompScriptWorld* script_world = (CompScriptWorld*)params.m_World;
+        update_result.m_TransformsUpdated = false;
```

Comment:
do not rise flag `update_result.m_TransformsUpdated` until at least one update function called